rasa_core/channels/slack.py

```diff
@@ -127,22 +140,30 @@ def _is_user_message(slack_event):
                 slack_event.get('event').get('text') and not
                 slack_event.get('event').get('bot_id'))
 
+    @staticmethod
+    def _is_interactive_message(payload):
+        return (payload['type'] == u"interactive_message")
```

Comment:
there should be no difference with or without `u` - (python 3 only)
```suggestion
        return (payload['type'] == "interactive_message")
```